sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0 (Unreleased)
 
+## 2020-07-10
+
+- [Bug Fix] Fixed an issue in browser recording where some recorded requests are not saved.
```

Comment:
```suggestion
- [Bug Fix] Fixed an issue where the browser-recording file is saved before all the recorded requests are pushed to the array of recordings(request-response pairs).
```
More info suggesting that the final requests of the test are not being saved or something in these lines?

Reply:
Looks good!  The missing ones are not necessarily the final/last requests though.  It depends on how long a request takes.